HealthcareSystem/PatientWebApp/wwwroot/css/style_for_unpublished_feedbacks.css

```diff
@@ -0,0 +1,131 @@
+﻿body#add-apart, body#guests-review, body#comments {
```

Comment:
As far as I can see you have Bootstrap included in the project, I feel like using that would be much easier than writing custom CSS?